core/domain/app_feedback_report_services_test.py

```diff
@@ -63,13 +63,13 @@ class AppFeedbackReportServicesUnitTests(test_utils.GenericTestBase):
     USER_ID = 'user_1'
     REPORT_TYPE_SUGGESTION = (
         app_feedback_report_constants.ReportType.SUGGESTION)
-    CATEGORY_OTHER = app_feedback_report_constants.CATEGORY.other_suggestion
+    CATEGORY_OTHER = app_feedback_report_constants.Category.OTHER_SUGGESTION
```

Comment:
In this file, there are several places where this enum is used in the form of `CATEGORY_OTHER.name`. Here's an example (line 163):

![image](https://user-images.githubusercontent.com/78612244/185728309-edec168b-6ad6-4a0f-86b9-8b90885ba131.png)

You will have to change these from `CATEGORY_OTHER.name` to `CATEGORY_OTHER.value` (I think there are 7-8 instances of this in this file).